styles/variables-silicon.less

```diff
@@ -10,12 +10,19 @@
 
 // Button
 // ---------------------------------------
-@agate-button-border-radius: 6px;
+@agate-button-border-radius: 37px;
 @agate-button-client-animation-offset: (@agate-button-animation-duration * 0.4);
 @agate-button-badge-animation-offset: (@agate-button-animation-duration * 0.8);
 @agate-button-badge-position: -3px 0 auto auto;
-@agate-button-small-height: 48px;
-@agate-button-small-font-size: 24px;
+@agate-button-border-width: 2px;
+@agate-button-font-size: 36px;
+@agate-button-font-weight: 300;
+@agate-button-height: 75px;
+@agate-button-line-height: 1.17;
+@agate-button-border-focus-width: 0px;
```

Comment:
Rather than setting the border-width here, it's safer to leave the border width alone, and set the border-color to transparent instead. Border is associated with DOM layout, like padding and margin, which can cause unexpected shifting when you change it, so it's best to just avoid it altogether.